README.md

```diff
@@ -22,7 +22,9 @@ def deps do
 end
 ```
 
-This package depends on [FDK-AAC](https://github.com/mstorsjo/fdk-aac) library.
+This package depends on [FDK-AAC](https://github.com/mstorsjo/fdk-aac) library. The precompiled 
+build will be pulled and linked automatically. However, should there be any problems, consider installing it manually.
+
 
 ### Ubuntu
```

Comment:
How about changing that header slightly, to emphasize that this is a case for manual installation? 
```suggestion
### Manual installation on Ubuntu
```